composer.json

```diff
@@ -11,11 +11,11 @@
         }
     ],
     "require": {
-        "cakephp/cakephp": "^3.0",
+        "cakephp/cakephp": "^4.0",
```

Comment:
I guess we could support both 3.0 and 4.0? If so, it should probably look like `^3.0|^4.0`